tests/evaluate/test_default_evaluator.py

```diff
@@ -2684,3 +2684,37 @@ def test_eval_df(predictions, targets, metrics, inputs, truth, context):
         )
 
     assert eval_df_value.equals(test_eval_df_value)
+
+
+def identity_model(inputs):
+    return inputs
+
+
+def test_default_metrics_as_custom_metrics():
+    with mlflow.start_run() as run:
+        model_info = mlflow.pyfunc.log_model(
+            artifact_path="model", python_model=identity_model, input_example=["a", "b"]
+        )
+        data = pd.DataFrame(
+            {
+                "question": ["words random", "This is a sentence."],
+                "answer": ["words random", "This is a sentence."],
+            }
+        )
+        mlflow.evaluate(
```

Comment:
Can we get the results = mlflow.evaluate

results.metrics has a dict of all the metrics. Can we assert that the metrics in the custom metrics are part of the result dictionary and not ignored